src/components/image/components.test-pw.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+import carbonLogo from "../../../logo/carbon-logo.png";
+
+import Image from ".";
+import useMediaQuery from "../../hooks/useMediaQuery";
+
+import Box from "../box";
+import Button from "../button";
+import Typography from "../typography";
+import flexibleSvg from "../../../.assets/flexible.svg";
+import pointSvg from "../../../.assets/point.svg";
+import brushSvg from "../../../.assets/brush.svg";
+import collaborateSvg from "../../../.assets/collaborate.svg";
+import { StyledImageProps } from "./image.style";
+
+export const CarbonLogoImage = (props: StyledImageProps) => (
```

Comment:
**comment(non-blocking)**: what do you think about using the `ImageProps` interface here? Just so we are using public facing exports in our test examples

Reply:
I'm OK with that as this was just the type interface VS Code inferred from it's usage. I'll get it changed 😄